Tests/MongoSwiftTests/CodecTests.swift

```diff
@@ -442,9 +442,13 @@ final class CodecTests: MongoSwiftTestCase {
     // test that Document.init(from decoder: Decoder) works with a non BSON decoder and that
     // Document.encode(to encoder: Encoder) works with a non BSON encoder
     func testDocumentIsCodable() throws {
+        // We presently have no way to control the order of emitted JSON in `cleanEqual`, so this
```

Comment:
Is there an SR ticket or bug report that tracks what's going wrong here? If so, it might be worth adding to the comment. Also might be worth filing a SWIFT ticket if there's work that needs to be done by us.

Reply:
They do provide a `.sortedKeys` option for `JSONSerialization.writeJSONObject`, but it only works on iOS, and iOS 10.13.x+ at that (and it doesn't seem to work on 10.14.2, afaict). The issue we are left with is that `[String: Any]` isn't sortable. I think we should figure out another way to test what's happening here. We already disabled the test on Linux because I imagine they incorporated changes earlier than OSX that were already on the way.

Reply:
This test can just be updated as part of [SWIFT-327](https://jira.mongodb.org/browse/SWIFT-327) / [SWIFT-328](https://jira.mongodb.org/browse/SWIFT-328). Until then it looks good as it is 👍